### PR TITLE
Permissions request

### PR DESCRIPTION
**Pull request by BartSoftware**

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -12,4 +12,10 @@
         <item name="android:elevation">4dp</item>
     </style>
 
+    <style name="AlertDialogStyle" parent="Theme.AppCompat.Light.Dialog.Alert">
+        <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:textColorPrimary">@color/colorPrimaryDark</item>
```
Comment (kejmil01) [inline app/src/main/res/values/styles.xml]: I think text should black or some-what-black.

**Comment by kejmil01** [comment]
You have to cover case when user selects "Do not ask again option" and then clicks on reject button.

What should happen after mentioned steps when user clicks on OK button in "Missing permissions" dialog?

Do you think it is ok to leave the user in looping permission dialog state? My suggestion is to move user to app's settings screen (android system settings) - you can find implementation of that in NearbyPlaces project i think (https://code.intive.com/stash/projects/SGE/repos/nearby-places/browse)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <resources>
     <string name="app_name">GoInvisible</string>
     <string name="unsupported_extension">Editing works only for JPEG images</string>
+    <string name="permissions_dialog_title">Missing permissions</string>
+    <string name="permissions_dialog_message">Permissions are necessary to use this application.</string>
```
Comment (kejmil01) [inline app/src/main/res/values/strings.xml]: Maybe smth like "Storage permission is necessary to use this application"?

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (BartSoftware) [reply app/src/main/res/values/styles.xml]: What do you mean? Should i precise separate color for text?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -2,6 +2,6 @@
     <string name="app_name">GoInvisible</string>
     <string name="unsupported_extension">Editing works only for JPEG images</string>
     <string name="permissions_dialog_title">Missing permissions</string>
-    <string name="permissions_dialog_message">Permissions are necessary to use this application.</string>
+    <string name="permissions_dialog_message">Storage permission is necessary to use this application</string>
```
Comment (kejmil01) [inline app/src/main/res/values/strings.xml]: In previous comment i forgot about dot at the end - sorry about that. So for me final version would be "Storage permission is necessary to use this application.".

**Review by kejmil01 — APPROVED** [review]

**Pull request merged**